### ISSUE TITLE
No music

### ISSUE THREAD

**Issue by lotdrops** [karimknaebel/phonograph-issue-tracker#3]
I installed latest version of marshmallow fresh (deleting everything), and phonograph  now doesn't load the music. In folders it has found the music but not in the library (couldn't find the refresh button). 
Also, in folders if I click a song sometimes a dialog appears for some milliseconds (can't read) and sometimes it doesn't, but in no case does it play music. 
Version 0.11.0RC

**Comment by karimknaebel** [comment]
Try using "scan" from the overflow menu in the folders. If this doesn't work please provide additional information (device name, music on sdcard or internal storage?).

**Comment by lotdrops** [comment]
Something very odd happened. I had created the folder with adb when transferring the files and they couldn't be found. When I copied them to another folder with a file Explorer the problem was solved. 
Other music players couldn't find the songs, but could play them. 
There is no scan in the overflow button. Shuffle all,  grid size, colored footers, sleep timer and equalizer. 

I have a stock nexus 5
